tests/standalone/run_all.py

```diff
@@ -264,6 +265,35 @@
     "MFCM140U.DLL",
 )
 
+# checks requirements needed to run each test module, according to the specified special comment
+# special comments are in the following formats:
+#     "# nuitka-skip-unless-expression: expression to be evaluated"
+#       OR
+#     "# nuitka-skip-unless-imports: module1,module2,..."
+def checkRequirements(filename):
+    with open(filename,'r') as f:
```

Comment:
One last quibble, can you make use of `getFileContentByLine` from `nuitka.Utils.FileOperations`, because that avoids the deep nesting here.

Reply:
Done. Anything else?